### PR TITLE
Link GH workflow & Crates.io badges

### PR DESCRIPTION
**Pull request by dinomite**
The GH workflow badge was just the raw image, clicking it loaded that image in a new tab.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a missed `flush` call that may be causing communication errors (#521)
 - Fix "SHA-256 comparison failed: [...] attempting to boot anyway..." (#567)
 - Windows: Update RST/DTR order to avoid issues.
+- Links to build & crate (#568)
```
Comment (SergioGasquez) [inline CHANGELOG.md:36]: Maybe we dont need to add an entry for this change

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # espflash
 
-![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/esp-rs/espflash/ci.yml?branch=main&labelColor=1C2C2E&logo=github&style=flat-square)
-![Crates.io](https://img.shields.io/crates/l/espflash?labelColor=1C2C2E&style=flat-square)
+[![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/esp-rs/espflash/ci.yml?branch=main&labelColor=1C2C2E&logo=github&style=flat-square)](https://github.com/esp-rs/espflash/actions/workflows/ci.yml)
+[![Crates.io](https://img.shields.io/crates/l/espflash?labelColor=1C2C2E&style=flat-square)](https://crates.io/crates/espflash)
```
Comment (SergioGasquez) [inline README.md:4]: Not sure if this should point to `espflash` on crates.io, as its the readme which contains info about `espflash` and `cargo-espflash`. Tbh, I am not sure if we should even have this badge at all, so Id rather keep it as it was. For the CI badge, I think adding the link is definitely an improvement!

**Review by SergioGasquez — COMMENTED** [review]
Thanks for the contribution! Left some minor comments/thoughs

File: README.md (hunk 2 above)
Comment (dinomite) [reply README.md:4]:
```suggestion
![Crates.io](https://img.shields.io/crates/l/espflash?labelColor=1C2C2E&style=flat-square)
```

**Review by SergioGasquez — APPROVED** [review]
LGTM!

**Pull request merged**